test/asserts/big-number-assert_test.js

```diff
@@ -22,29 +22,77 @@ const Assert = BaseAssert.extend({
 describe('BigNumberAssert', () => {
   it('should throw an error if the input value is not a big number', () => {
     const choices = [[], {}, ''];
+    const options = [undefined, { validateSignificantDigits: true }, { validateSignificantDigits: false }];
```

Comment:
If all options are the same, WDYT about moving them out of individual tests? Maybe something like:

```js
[undefined, { validateSignificantDigits: true }, { validateSignificantDigits: false }].forEach(options => {
  describe(`with options '${options}'`, () => {
    it('should throw an error if the input value is not a big number', () => {
      [[], {}, ''].forEach(choice => {
        try {
          new Assert().BigNumber(options).validate(choice);

          // ...
        }
      });
    });

    // ...
  });
});
```

Reply:
I had the same ideia before and actually began that refactor, but I was doing it without the extra `describe`, which made change every `it` name to include the `option`... so I kinda gave up 😄 

I'll try it out and update the PR.

Reply:
Done 👍